### PR TITLE
Updated German translation

### LINKED ISSUE #26: Update German Translation
After the update the German Translation should be updated I think.

### PR DESCRIPTION
**Pull request by LiFaytheGoblin**
fixes #26

### CODE REVIEW

File: i18n/de.json [hunk 1]
```diff
@@ -32,22 +33,28 @@
...
 	"scratchwikiskin-footer-community-title": "Gemeinschaft",
 	"scratchwikiskin-footer-cgs": "Regeln",
 	"scratchwikiskin-footer-discuss": "Diskussionsforen",
 	"scratchwikiskin-footer-wiki": "Scratch Wiki",
 	"scratchwikiskin-footer-stats": "Statistiken",
-	"scratchwikiskin-footer-support-title": "Unterst\u00fctzung",
+	"scratchwikiskin-footer-cp": "Community Portal",
+	"scratchwikiskin-footer-support-title": "Scratch-Wiki:Hilfe:Inhaltsverzeichnis",
+	"scratchwikiskin-footer-help-page-url": "Help:Contents",
++	"scratchwikiskin-footer-help": "Hilfe für Autoren",
```
Comment (Kenny2github) [inline i18n/de.json]: Random extra +?

File: i18n/de.json [hunk 2]
```diff
@@ -32,22 +33,28 @@
...
 	"scratchwikiskin-footer-tips": "Tipps",
 	"scratchwikiskin-footer-faq": "FAQ",
 	"scratchwikiskin-footer-offline": "Offline Editor",
-	"scratchwikiskin-footer-contact": "Kontaktiere das Scratch Team",
 	"scratchwikiskin-footer-store": "Scratch Laden",
-	"scratchwikiskin-footer-donate": "Spenden",
+	"scratchwikiskin-footer-donate": "An Scratch spenden",
++	"scratchwikiskin-footer-legal-title": "Rechtliches",
++	"scratchwikiskin-footer-privacy": "Datenschutz",
++	"scratchwikiskin-footer-dmca": "Impressum",
```
Comment (Kenny2github) [inline i18n/de.json]: Ditto

Comment (LiFaytheGoblin) [reply i18n/de.json]: huh super weird.

**Comment by LiFaytheGoblin** [comment]
Check again? @Kenny2github

**Pull request merged**